mkat_fpga_tests/test_cbf.py

```diff
@@ -59,15 +59,18 @@ def setUp(self):
         self.addCleanup(self.corr_fix.stop_x_data)
         self.corr_fix.start_x_data()
         self.corr_fix.issue_metadata()
+        self.test_chan=1500
+        self.requested_test_freqs = self.corr_freqs.calc_freq_samples(
+            self.test_chan, samples_per_chan=101, chans_around=5)
+        self.expected_fc = self.corr_freqs.chan_freqs[self.test_chan]
+        # Consistency check threshold
+        self.threshold=0.01
```

Comment:
This  is probably a bit big (equates to -20 dB). We probably want to test down to -70 dB, so make it 1e-7, and add a comment pointing out that it is -70dB.